Maestro/subscriptions.json

```
@@ -793,14 +793,21 @@
         "https://github.com/dotnet/diagnostics/blob/master/**/*",
         "https://github.com/dotnet/machinelearning/blob/master/**/*",
         "https://github.com/dotnet/machinelearning/blob/release/**/*",
+        "https://github.com/dotnet/roslyn/blob/dev/**/*",
+        "https://github.com/dotnet/roslyn/blob/features/**/*",
+        "https://github.com/dotnet/roslyn/blob/master**/*",
```

Maybe you're not done, but noticed:

```suggestion
        "https://github.com/dotnet/roslyn/blob/master/**/*",
```